### PR TITLE
UGENE-7077. Unify code style for functions in header files in biostru…

### PR DESCRIPTION
**Pull request by yalgaer**

### CODE REVIEW

**Review by atiunov — DISMISSED** [review]

File: src/plugins/biostruct3d_view/src/BioStruct3DGLRender.h [hunk 1]
```diff
@@ -152,27 +150,19 @@ class BioStruct3DGLRenderer {
     void setColorScheme(const BioStruct3DColorScheme *s);
 
     //! @returns current color scheme.
-    const BioStruct3DColorScheme *getColorScheme() const {
-        return colorScheme;
-    }
+    const BioStruct3DColorScheme *getColorScheme() const;
```
Comment (evelinabis) [inline src/plugins/biostruct3d_view/src/BioStruct3DGLRender.h:153]: The definition is lost. Also getShownModelsIndexes, setShownModelsIndexes, isAvailableFor

File: src/plugins/biostruct3d_view/src/BioStruct3DSplitter.h [hunk 2]
```diff
@@ -169,9 +167,7 @@ class DBLinksFile {
 
 public:
     bool load();
-    QList<DBLink> getLinks() {
-        return links;
-    }
+    QList<DBLink> getLinks();
```
Comment (evelinabis) [inline src/plugins/biostruct3d_view/src/BioStruct3DSplitter.h:170]: The definition is lost

**Review by evelinabis — CHANGES_REQUESTED** [review]

**Review by evelinabis — APPROVED** [review]

File: src/plugins/biostruct3d_view/src/BioStruct3DGLRender.h (hunk 1 above)
Comment (yalgaer) [reply src/plugins/biostruct3d_view/src/BioStruct3DGLRender.h:153]: Good job!

**Review by DmitriiSukhomlinov — APPROVED** [review]
Please, add a "Refactoring" label

**Pull request merged**